examples/ios/objc/Draw/Models/DrawPath.m

```diff
@@ -0,0 +1,52 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "DrawPath.h"
+
+@implementation DrawPath
+
+@dynamic path;
+
+- (UIBezierPath *)path
+{
+    UIBezierPath *path = [UIBezierPath bezierPath];
+    path.lineWidth = 4.0f;
+    NSUInteger index = 0;
+    for (DrawPoint *point in self.points) {
+        CGPoint cgPoint = CGPointMake(point.x, point.y);
+        if (index == 0) {
+            [path moveToPoint:cgPoint];
+        } else {
+            [path addLineToPoint:cgPoint];
+        }
+        index++;
+    }
+    return path;
+}
+
++ (NSDictionary *)defaultPropertyValues
+{
+    return @{ @"drawerID" : @"" };
+}
+
++ (NSArray *)ignoredProperties
```

Comment:
this shouldn't be necessary given that `path` is specified as a `readonly` property.